index.html

```diff
@@ -42,6 +42,202 @@ <h4> I’am a Software Developer</h4>
+                </ul>
+            </div>
+            <a class="see_project2" href="#">See Project</a>
+        </section>
+
+        <section class="card1">
+            <h2>Profesional Art Printing Data</h2>
+            <p class="jobs_p">
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                industry's standard
+            </p>
+            <div class="skills1">
+                <ul>
+                    <li>html</li>
+                    <li>javascript</li>
+                    <li>ruby</li>
+                </ul>
+            </div>
+            <a class="see_project2" href="#">See Project</a>
+        </section>
+
+        <section class="card1">
+            <h2>Profesional Art Printing Data</h2>
+            <p class="jobs_p">
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                industry's standard
+            </p>
+            <div class="skills1">
+                <ul>
+                    <li>html</li>
+                    <li>javascript</li>
+                    <li>ruby</li>
+                </ul>
+            </div>
+            <a class="see_project2" href="#">See Project</a>
+        </section>
+
+        <section class="card1">
+            <h2>Profesional Art Printing Data</h2>
+            <p class="jobs_p">
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                industry's standard
+            </p>
+            <div class="skills1">
+                <ul>
+                    <li>html</li>
+                    <li>javascript</li>
+                    <li>ruby</li>
+                </ul>
+            </div>
+            <a class="see_project2" href="#">See Project</a>
+        </section>
+
+        <section class="card1">
+            <h2>Profesional Art Printing Data</h2>
+            <p class="jobs_p">
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                industry's standard
+            </p>
+            <div class="skills1">
+                <ul>
+                    <li>html</li>
+                    <li>javascript</li>
+                    <li>ruby</li>
+                </ul>
+            </div>
+            <a class="see_project2" href="#">See Project</a>
+        </section>
+
+    </section>
+
+    <br>
+    <br>
+    <br>
+    <br>
+    <br>
+    <br>
+    <br>
+
+    <section class="container about_container">
+        <h1 class="text1">About Me</h1>
+        <p>
+            Hello I’m a software developer! I can help you build a product , feature or website Look through some of my
+            work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact
+            me.
+        </p>
+        <a class="see_project3" href="#">Get My Resume</a>
```

Comment:
- [ ] Please follow the button styling for: Active, Hover and Pressed, see screenshot below ⬇️ 


![uuuuuuu](https://user-images.githubusercontent.com/69638013/159150728-bf567afc-f07c-4b05-95b9-e1403a7c6d38.png)